internal/backend/remote-state/s3/backend.go

```diff
@@ -164,13 +166,6 @@ func New() backend.Backend {
 				Default:     false,
 			},
 
-			"skip_metadata_api_check": {
-				Type:        schema.TypeBool,
-				Optional:    true,
-				Description: "Skip the AWS Metadata API check.",
-				Default:     false,
-			},
-
```

Comment:
I removed this, as it seems to be irrelevant as far as configuring a s3 backend goes.

Reply:
I'm not @hashicorp/terraform-aws, but it's worth noting that changing the schema for backends can cause users supplying backend configuration via variables to need to reconfigure the backend: https://github.com/hashicorp/terraform/issues/32866#issuecomment-1471921440
We therefore tend to be conservative in changing backend schemas.